hugo/content/docs/micro-layout.md

```diff
@@ -6,133 +6,335 @@ weight: 400
 The **micro-layout** refers to the layout of elements inside of a node, i.e. the layout of nested labels, buttons, etc. It is not to be confused with the **macro-layout** which refers to the layout of the entire graph, e.g. the position of nodes.
 
 ## Layout Configuration
-Each element inheriting from `SNode` or `SCompartment` can have a property `layout` that specifies the layout of its children elements. Sprotty implements three layout configurations by default:
+
+Any model element that implements or extends the `SNode` or `SCompartment` interface has an optional property `layout` that specifies the general layout of its children elements. Sprotty implements three layout configurations by default:
+
+* `stack`: children elements are stacked on top of each other (default layout)
 * `hbox`: children elements are arranged horizontally
 * `vbox`: children elements are arranged vertically
-* `stack`: children elements are stacked on top of each other
 
-The [Class Diagram example](https://github.com/eclipse-sprotty/sprotty/tree/master/examples/classdiagram/) is a good starting point for understanding the layout configuration.
-Let's focus on `node0` which represents the class `Foo`:
+The `layout` property aims at arranging children elements that do not have a meaning in terms of graph hierarchy (i.e. labels, buttons, ...). Please note that nested `SNodeImpl` do not respect the `layout` property by default (more on that later).
```

Comment:
Please note that a child [bound to SNodeImpl](/docs/dependency_injection/#the-container) does not respect...
...(more on that [later](https://1313-gfontorbe-sprottywebsit-cbdcvwsc8az.ws-eu104.gitpod.io/docs/micro-layout/#layouting-nested-nodes)).